python/pyspark/sql/dataframe.py

```diff
@@ -861,11 +861,11 @@ def groupBy(self, *cols):
             Each element should be a column name (string) or an expression (:class:`Column`).
 
         >>> df.groupBy().avg().collect()
-        [Row(avg(age)=3.5)]
+        [Row(avg_c1=3.5)]
```

Comment:
@cloud-fan 
val df = Seq(1, 2, 3).map(i => (i, i.toString)).toDF("int", "str")
val df2 = df.as('x).join(df.as('y), $"x.str" === $"y.str").groupBy("y.str").max("y.int")
intercept[Throwable](df2.write.parquet(filePath))

I am reading it as though, we don't like the alias names that is generated by us (system picked) ? Shouldn't this write go through without any problem ? When i changed the alias name to a simpler one like max_c0 instead of max(c0) , this test started failing and thats how i came to know about it.